.github/workflows/check-dependencies.yml

```diff
@@ -0,0 +1,32 @@
+name: Check Dependencies
+
+on:
+  schedule:
+    - cron: '00 0 * * *'
+
+jobs:
+  check-language-servers:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package:
+          - salesforcedx-vscode-lwc
+          - salesforcedx-vscode-lightning
+    steps:
+      - uses: actions/checkout@master
+      - name: set remote url
+        run: git remote set-url --push origin https://$GITHUB_ACTOR:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY
+      - name: package-update
+        uses: taichi/actions-package-update@master
```

Comment:
how much work is it to do something similar to this ourselves? I'd prefer not to use a 3rd party github action.

If we do use this we need to get a security review and we should also point to a specific tag and not @`master`, which I'm assuming is dynamic. If we could do something ourselves though I'd lean towards that. It doesn't have to be fancy.

Reply:
I have the same concerns as you.  I don't think it would take more than a couple days to create a similar action.

If we decide to use this 3rd party action, I agree that we should set a specific version.